python/grass/jupyter/display.py

```diff
@@ -150,7 +150,7 @@ def __getattr__(self, name):
         grass_module = name.replace("_", ".")
         # Assert module exists
         if not shutil.which(grass_module):
-            raise AttributeError(_("Cannot find GRASS module {}").format(grass_module))
+            raise AttributeError(_(f"Cannot find GRASS module {grass_module}"))
```

Comment:
This message and the ones in `render3d.py` are actually translatable (surrounded with `_(...)`), so f-string won't work well here because it will fail to translate. You could   change these to not translated to make it consistent with the rest (where we intentionally ignored the issue of translating to avoid the need to translate messages in experimental code) However, since these are already translatable, you can keep them and that will be consistent with the rest of GRASS code.